### PR TITLE
Update seeds

### PR DESCRIPTION
**Pull request by pezholio**
This updates the seed scripts to be more performant with larger datasets and also ignore some old/incorrect data.

### CODE REVIEW

File: db/seeds/places.rb [hunk 1]
```diff
@@ -8,12 +8,27 @@
...
 )
 
+duplicates_filename = File.join(
+  Rails.root,
+  "db",
+  "data",
+  "missing_photos.csv"
+)
+
+duplicates = CSV.parse(
```
Comment (CristinaRO) [inline db/seeds/places.rb]: The name `duplicates` confuses me. Could you rename it to better reflect its purpose, e.g. `inactive_images`?

File: db/seeds/votes.rb [hunk 2]
```diff
@@ -8,13 +8,16 @@
...
     place_id: row["place"],
     uuid: row["uuid"],
     rating: row["rating"],
     created_at: DateTime.parse(row["date_submitted"])
   }
-
-  Vote.find_or_create_by(id: row["id"].to_i).update!(params)
 end
+
+Vote.upsert_all(rows.uniq { |vote| [vote[:place_id], vote[:uuid]] }, unique_by: :id)
```
Comment (CristinaRO) [inline db/seeds/votes.rb]: If I understand this right, we're eliminating any accidental duplicate votes in Ruby, before we even pass them to the `upsert`?

Comment (pezholio) [reply db/seeds/votes.rb]: Yeah, that's right 👍

Comment (pezholio) [reply db/seeds/votes.rb]: I've made that into a variable of its own to make that clearer now

**Review by CristinaRO — APPROVED** [review]

**Pull request merged**